src/main/java/org/galatea/starter/service/IexService.java

```diff
@@ -95,6 +99,19 @@ else if (date != null) {
     }
   }
 
+  /**
+   * Check if the date is a weekend.
+   * @param date date (string formatted YYYYMMDD) we are considering
+   * @return boolean: true if the date is a weekend, false otherwise
+   */
+  public boolean isDateWeekend(final String date) {
+    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyyMMdd");
```

Comment:
The formatter here is constant, it should be declared at the top of the class using the Java standard `private static final Object OBJECT_NAME;` convention instead of creating it each time.

Reply:
Done!